packages/core/src/base-chart.ts

```diff
@@ -51,19 +51,20 @@ export class BaseChart {
 	constructor(holder: Element, configs: any) {
 		this.id = `chart-${BaseChart.chartCount++}`;
 
-		(holder as HTMLElement).style.position = "relative";
+		if (configs.options) {
+			this.options = Object.assign({}, this.options, configs.options);
+		}
+
+		// Save holder element reference, and initialize it by applying appropriate styling
 		this.holder = holder;
+		this.styleHolderElement();
 
 		const {chartId, container} = this.setChartIDContainer();
 		this.container = container;
 		this.chartContainerID = chartId;
 
-		if (configs.options) {
-			this.options = Object.assign({}, this.options, configs.options);
-
-			if (this.options.containerResizable) {
-				this.resizeWhenContainerChange();
-			}
+		if (this.options && this.options.containerResizable) {
```

Comment:
Based on what I see here `this.options` should always exist and checking for it is redundant.

Reply:
Good catch 👍